Site/ASAB Maestro/Descriptors/seacat-auth.yaml

```diff
@@ -55,6 +55,77 @@ asab:
       # TODO ^^ this is not possible now
       host: <mocked>
 
+asab-config:
+  seacat-auth:
+    __schema__:
+      file: {}
+      if_not_exists: true
```

Comment:
This should be a default - I want to upload **schema** into the asap-config "as frequently as possible".

Reply:
so ...  if_not_exists: false for schema, because we want to update it
if_not_exists: true for config because this will be uploaded only if it does not exists at all